### DISCUSSION TITLE
Communicate with other process by add scripts

### DISCUSSION THREAD

**Discussion by cxdinter** [zeek/zeek#2082] [Archive]
Hi,
   I want to transfer some log information to other process when some events happen.
   For example, when Zeek detected the SSH login, is it possible to transfer the information to other process immediately by adding scripts ? (Currently, Zeek just record info into log file)
   Is there any guide for this requirement ? Thank you in advance.

**Comment by 0xxon** [comment]
Yes, this is possible using broker.

https://docs.zeek.org/en/master/frameworks/broker.html is a starting point in the documentation; however, it might be easiest to just see how some of the base scripts do this.
